src/compactor.rs

```diff
@@ -499,6 +499,7 @@ mod tests {
             compression_codec: None,
             object_store_cache_options: ObjectStoreCacheOptions::default(),
             block_cache_options: None,
```

Comment:
Can you remove the `block_cache_options` as part of this PR? Since we're providing an instance, we don't need the options anymore.

Reply:
yeah I think it can be replaced with no_block_cache, just like RocksDB's API and then if the instance is None, we create a default block cache.

Reply:
I've removed this field.